### PR TITLE
Update Profile screen, cleanup

### PR DESCRIPTION
**Pull request by kenan-mamedoff**

### CODE REVIEW

File: brd-ios/breadwallet/src/Redesign/Core/Constants/Margins.swift [hunk 1]
```diff
@@ -27,6 +27,8 @@ enum Margins: CGFloat {
     case huge = 24
     /// 32
     case extraHuge = 32
+    /// 36
+    case extraExtraHuge = 36
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Core/Constants/Margins.swift:31]: do we still need the extraHuge? :D

File: brd-ios/breadwallet/src/Redesign/Scenes/Profile/ProfileViewController.swift [hunk 2]
```diff
@@ -59,28 +58,31 @@ class ProfileViewController: BaseTableViewController<ProfileCoordinator,
...
+//                    config = Presets.InfoView.pending
+//
+//                case (.levelTwo, .levelTwo(.resubmit)),
+//                    (.levelTwo, .levelTwo(.declined)),
+//                    (.levelTwo, .levelTwo(.expired)):
+//                    config = Presets.InfoView.declined
+//
+//                default:
+//                    config = Presets.InfoView.verification
+//                }
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Scenes/Profile/ProfileViewController.swift:82]: needed?

**Review by rcresnik — APPROVED** [review]

File: brd-ios/breadwallet/src/Redesign/Core/Constants/Margins.swift (hunk 1 above)
Comment (kenan-mamedoff) [reply brd-ios/breadwallet/src/Redesign/Core/Constants/Margins.swift:31]: yes. for navigation items.

File: brd-ios/breadwallet/src/Redesign/Scenes/Profile/ProfileViewController.swift (hunk 2 above)
Comment (kenan-mamedoff) [reply brd-ios/breadwallet/src/Redesign/Scenes/Profile/ProfileViewController.swift:82]: let's keep it for now.

**Pull request merged**